PubNub/Data/Managers/PNPublishSequence.m

```diff
@@ -369,30 +376,6 @@ - (void)subscribeOnNotifications {
 #endif // TARGET_OS_OSX
 }
 
-- (void)unsubscribeFromNotifications {
```

Comment:
Maybe it's the merge conflicts? Did you notice there are merge conflicts after I merged the bug fix?